roles/kargo-defaults/defaults/main.yaml

```diff
@@ -97,11 +97,15 @@ kube_apiserver_insecure_port: 8080 # (http)
 # Path used to store Docker data
 docker_daemon_graph: "/var/lib/docker"
 
+# Docker log options
+# Rotate container stderr/stdout logs at 50m and keep last 5
+docker_log_opts: "--log-opt max-size=50m --log-opt max-file=5"
```

Comment:
Let's update documentation. There are 2 cases
1. Operator has own logrotate, so he wants to disable it
2. Operator wants to increase/decrease size/number of files